config/Sage/config.txt

```diff
@@ -517,6 +516,10 @@ TOPIC_MAX_SIZE 500
 ## Uncomment to enable Fail2Topic, comment-out to disable
 #TOPIC_ENABLED
 
+## Uncomment to enable global ban DB using the provided URL. The API should expect to receive a ckey at the end of the URL.
+## More API details can be found here: https://centcom.melonmesa.com/swagger/index.html
+#CENTCOM_BAN_DB https://centcom.melonmesa.com/ban/search/
```

Comment:
Best practice would be to remove the trailing slash here and instead add it in later